api/src/test/java/org/pmiops/workbench/actionaudit/targetproperties/WorkspaceTargetPropertyTest.kt

```diff
@@ -64,7 +61,7 @@ class WorkspaceTargetPropertyTest {
                 WorkspaceTargetProperty.values(),
                 workspace1!!)
 
-        assertThat(propertiesByName).hasSize(6)
+        assertThat(propertiesByName).hasSize(20)
```

Comment:
That's a good idea. I had no end of pain dealing with squirrelly initialization and translation behavior in testing. 

About this point I started going down a rabbit hole on sensible initialization for API models, avoiding null booleans and lists, since Swagger just gives a very naive default constructor with no way to override it. See https://github.com/all-of-us/workbench/pull/2884 and https://github.com/all-of-us/workbench/pull/2888.